firefox-ios/Client/Frontend/Library/HistoryPanel/HistoryPanel.swift

```diff
@@ -585,6 +592,7 @@ class HistoryPanel: UIViewController,
             view.backgroundColor = self.currentTheme().colors.layer1
```

Comment:
I went up a couple lines, do the comments on 589 and 590 need to be modified or deleted?